apps/src/music/player/MusicLibrary.ts

```diff
@@ -165,11 +165,13 @@ export default class MusicLibrary {
   // Determine the available sound types available in this library.
   // Only currently-allowed sounds from packs are included.
   private determineAvailableSoundTypes() {
-    const folders = this.getAllowedSounds();
+    const folders = this.getAllowedSounds() as SoundFolderPack[];
```

Comment:
Could `getAllowedSounds()` return a `SoundFolderPack[]`? I don't think allowed sounds applies to instruments/kits right?